test/webpack.js

```diff
@@ -19,6 +19,11 @@ export default function ({ fixture = 'basic.js', options, extend = {} } = {}) {
       }],
     },
     ...extend,
+    plugins: [
+      new webpack.optimize.CommonsChunkPlugin({
```

Comment:
Yup, you can see the change in the [snapshot diff](https://github.com/webpack-contrib/istanbul-instrumenter-loader/pull/66/files#diff-26e4d9ebc388064364a6cf68d73f91f8), its much cleaner 😄